cost-analyzer/templates/_helpers.tpl

```diff
@@ -51,15 +51,21 @@ Kubecost 2.0 preconditions
 
   {{/*https://github.com/helm/helm/issues/8026#issuecomment-881216078*/}}
   {{- if ((.Values.thanos).store).enabled -}}
-  {{- fail "Kubecost no longer includes Thanos by default. Please see https://docs.kubecost.com/install-and-configure/install/thanos for more information." -}}
+  {{- fail "\n\nKubecost no longer includes Thanos by default. Please see https://docs.kubecost.com/install-and-configure/install/thanos for more information." -}}
   {{- end -}}
 
   {{- if (.Values.federatedETL).primaryCluster -}}
-    {{- fail "In Kubecost 2.0, there is no such thing as a federated primary. If you are a Federated ETL user, this setting has been removed. Make sure you have kubecostAggregator.deployMethod set to 'statefulset' and federatedETL.federatedCluster set to 'true'." -}}
+    {{- fail "\n\nIn Kubecost 2.0, there is no such thing as a federated primary. If you are a Federated ETL user, this setting has been removed. Make sure you have kubecostAggregator.deployMethod set to 'statefulset' and federatedETL.federatedCluster set to 'true'." -}}
+  {{- end -}}
+
+  {{- if or (.Values.saml.rbac).enabled (.Values.oidc.rbac).enabled -}}
```

Comment:
```suggestion
  {{- if or ((.Values.saml).rbac).enabled ((.Values.oidc).rbac).enabled -}}
```

I don't think the `()` are needed in this case, because all these values are defined in the values.yaml file. But if we do want to perform these safety checks, I _think_ we have to modify slightly